examples/basic/main.tf

```diff
@@ -0,0 +1,19 @@
+data "aws_ecs_cluster" "example" {
+  cluster_name = "example"
+}
+
+data "aws_vpc" "initial" {}
+
+data "aws_vpc" "operations" {}
+
+module "traefik" {
+  source            = "../../"
+  name              = "traefik"
+  aws_region        = var.aws_region
```

Comment:
does this example module call work without manually supplying variables on the cli? 🤔 

If it does not, then it should adjusted it in way that it works easily.